docs/polaris/programming-models/sycl-polaris.md

```diff
@@ -83,7 +83,7 @@ int main(int argc, char *argv[]){
   int thread_id = 0;
 
   if(num_devices == 0){
-#pragma omp parallel default(shared) private(hwthread, thread_id)
+#pragma omp parallel
```

Comment:
how I mean do you still need omp ? I was thinking it was a port to SYCL. My bad!

Reply:
the snippet wasn't intended just for SYCL. It was a small code to show CPU-GPU binding-info when MPI, OpenMP and SYCL are compiled together. (i.e., how to compile. not really about the code)